auth/src/main/java/com/firebase/ui/auth/ui/idp/AuthMethodPickerActivity.java

```diff
@@ -101,6 +103,9 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         mHandler = ViewModelProviders.of(this).get(SocialProviderResponseHandler.class);
         mHandler.init(params);
 
+       if (params.lockOrientation) {
```

Comment:
Can this be moved into `AppCompatBase` so that we don't have to repeat it in each subclass?